From 261fbb28438f40c34df6f9a6cca7934448725efc Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@freescale.com>
Date: Mon, 9 Mar 2015 14:04:09 +0800
Subject: [PATCH 0102/5242] MLK-10387-2 ARM: imx7d-sdb: add USB support

commit  eaebdd9d9a8bdace09a966b146b26479a77cb64f from
https://source.codeaurora.org/external/imx/linux-imx.git

Add USB support for both USBOTG1 and USBOTG2.

Signed-off-by: Peter Chen <peter.chen@freescale.com>

Fixed for LPSR pin rename in 4.14

Signed-off-by: Leonard Crestez <leonard.crestez@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts |   13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index 857a223..ff574e9 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -375,6 +375,19 @@
 	status = "okay";
 };
 
+&iomuxc_lpsr {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_hog_2>;
+
+	imx7d-sdb {
+		pinctrl_hog_2: hoggrp-2 {
+			fsl,pins = <
+				MX7D_PAD_LPSR_GPIO1_IO05__GPIO1_IO5	0x14
+			>;
+		};
+	};
+};
+
 &uart1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_uart1>;
-- 
1.7.9.5

